<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditcardform</class>
 <widget class="QDialog" name="creditcardform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>199</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>289</width>
    <height>199</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>289</width>
    <height>199</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Payment Entry</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>211</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>211</height>
    </size>
   </property>
   <widget class="QLabel" name="FirstNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="LastNameLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstNamelineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="inputMask">
     <string notr="true">AAAAAAAAAAAAAAAAAAAAAAAAA; </string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastNamelineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true">AAAAAAAAAAAAAAAAAAAAAAAAA; </string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="cursorPosition">
     <number>25</number>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="CreditCardLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit Card Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CreditCardlineEdit_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true">9999; </string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="ExpDateLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Expiration Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="MonthLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLabel" name="YearLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="MonthspinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="YearspinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2011</number>
    </property>
    <property name="maximum">
     <number>2016</number>
    </property>
   </widget>
   <widget class="QPushButton" name="SubmitpushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearpushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="SecurityCodeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Security Code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SecurityCodelineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Your security code can be found at the end of the signature strip on the back of your card.</string>
    </property>
    <property name="statusTip">
     <string>Your security code can be found at the end of the signature strip on the back of your card.</string>
    </property>
    <property name="whatsThis">
     <string>Your security code can be found at the end of the signature strip on the back of your card.</string>
    </property>
    <property name="inputMask">
     <string>999; </string>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>1317789508_credit_cards.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="CreditCardlineEdit_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true">9999; </string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="CreditCardlineEdit_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true">9999; </string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="CreditCardlineEdit_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true">9999; </string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="cursorPosition">
     <number>4</number>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>FirstNamelineEdit</tabstop>
  <tabstop>LastNamelineEdit</tabstop>
  <tabstop>CreditCardlineEdit_1</tabstop>
  <tabstop>CreditCardlineEdit_2</tabstop>
  <tabstop>CreditCardlineEdit_3</tabstop>
  <tabstop>CreditCardlineEdit_4</tabstop>
  <tabstop>MonthspinBox</tabstop>
  <tabstop>YearspinBox</tabstop>
  <tabstop>SecurityCodelineEdit</tabstop>
  <tabstop>SubmitpushButton</tabstop>
  <tabstop>ClearpushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
